ui/components/app/modals/customize-nonce/customize-nonce.component.js

```diff
@@ -27,7 +27,9 @@ const CustomizeNonce = ({
   updateCustomNonce,
   getNextNonce,
 }) => {
-  const [customNonce, setCustomNonce] = useState('');
+  const defaultNonce =
```

Comment:
Minor, but if there's no `customNonceValue` and `nextNonce` isn't a number, then will the `defaultNonce` be set to `false`?

Reply:
Yes, although since I only moved and didn't change this expression, I don't expect that to happen. Even though we don't specifically type check that, `nextNonce` seems to be a number everywhere in the codebase.

I am not sure which default nonce number would improve this expression, so unless we discover a bug I would keep this as it is for now.